data_models.rb

```diff
@@ -0,0 +1,21 @@
+require 'data_mapper'
+
+if ENV['RACK_ENV'] != 'production'
+  require 'dotenv'
+  Dotenv.load('.env')
+end
+
+DataMapper.setup(:default, ENV['DATABASE_URL'])
```

Comment:
You use a different database in development, testing, and production.  This is the name of the ENV var heroku uses



--

Jesse Farmer (via iPhone)

http://codeunion.io · @jfarmer

jesse@codeunion.io · 773.319.9355

On Wed, Feb 18, 2015 at 5:15 PM, ric-reyes <notifications@github.com>
wrote:

>> @@ -0,0 +1,21 @@
>> +require 'data_mapper'
>> +
>> +if ENV['RACK_ENV'] != 'production'
>> +  require 'dotenv'
>> +  Dotenv.load('.env')
>> +end
>> +
>> +DataMapper.setup(:default, ENV['DATABASE_URL'])
> @openspectrum: I got this from the can-i-eat-here app. Why store the database URL in `.env`?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/codeunion/air-stable/pull/1/files#r24958702